File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function (a) {
    let x =0;
    for(i=0;i<a.length;i++){
        if(a[i]>a[x]) {
            x+=1;
            a[x]=a[i];
        }
    }
    return x+1;

};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] a) {

        int x = 0;
        for(int i=0;i<a.length;i++) {
            if(a[i]>a[x]) {
                x++;
                a[x] = a[i];
            }
        }
        return x+1;
        
    }
}